awx/ui/src/screens/InstanceGroup/ContainerGroupEdit/ContainerGroupEdit.js

```diff
@@ -21,9 +21,9 @@ function ContainerGroupEdit({ instanceGroup }) {
     result: initialPodSpec,
   } = useRequest(
     useCallback(async () => {
-      const { data } = await InstanceGroupsAPI.readOptions();
-      return data.actions.POST.pod_spec_override.default;
-    }, []),
+      const { data } = await InstanceGroupsAPI.readInstanceGroupOptions(instanceGroup.id);
+      return data.actions.PUT.pod_spec_override.default;
```

Comment:
:+1:  Nice. I understood the problem this addresses - that the UI will equate the ability to add an instance group to the ability to edit an instance group (and its pod spec). This is the form of solution I was anticipating for that problem, and is ideal. Now permissions are specific to the object, so assumptions have to change.